src/components/Pages/PlaylistCollection/Page.js

```diff
@@ -125,6 +125,15 @@ const PlaylistCollectionPage = ({ collection, nextLink = null, prevLink = null,
 
   const filterOutUnits = items.map(item => item.unit).filter(u => !!u) || [];
 
+  // Don't recommend lesson preparation, skip to next unit.
+  let recommendUnit = unit;
+  if (recommendUnit.id === Object.entries(collection.ccuNames).find(([unitUid, ccuName]) => ccuName === '0')[0]) {
+    const indexOfUnit = playlist?.items.findIndex((item) => item.id === unit.id) ?? -1;
```

Comment:
If playlist is undefined then you'll call `undefined.findIndex` and I'm not sure it will work...

Reply:
Good point. Added a check.